features/src/main/feature/camel-features.xml

```diff
@@ -935,10 +935,11 @@
 
 <!--        Please refer to the component page (https://camel.apache.org/ignite.html) and to the Ignite docs (https://apacheignite.readme.io/docs/osgi-installation-in-karaf#preparatory-steps) for more information.]]>-->
 <!--        </details>-->
```

Comment:
Please either uncomment the details or remove them

Reply:
uncommented, thanks